src/test/mod.rs

```diff
@@ -55,7 +62,6 @@ fn database_url_from_env(backend_specific_env_var: &str) -> String {
     }
 }
 
-
 const DB_URL: &str = "oci://\"diesel\"/diesel@//192.168.2.81:1521/orcl";
```

Comment:
We should not hardcode our internal ip here :wink:

Reply:
In general I do agree with you, but it's already out, second this is a VM/docker which just exists for that one task. So there is no harm anyway. Removed it, please remember to set the environment variable `OCI_DATABASE_URL` from now on.

Reply:
It's not about that this url is secret, it is just about that other people could test this crate without setting up a database at a specific ip :wink: